.github/workflows/licence-checker.yml

```diff
@@ -22,6 +22,27 @@ jobs:
           token: ${{ github.token }}
           config: .licenserc.yaml
 
+  dependency-license:
```

Comment:
can we rename to `check-dependency-license` and also rename `check-license` to `check-source-license`